lib/matplotlib/backends/web_backend/mpl.js

```diff
@@ -402,7 +419,8 @@ mpl.figure.prototype.mouse_event = function(event, name) {
     var x = canvas_pos.x;
     var y = canvas_pos.y;
 
-    this.send_message(name, {x: x, y: y, button: event.button});
+    this.send_message(name, {x: x, y: y, button: event.button,
+                             step: event.step});
```

Comment:
what done js do if you have not set a value for `event.step`?  I assume magically pass a null-like value...